features/steps/gradle_plugin_steps.rb

```diff
@@ -87,18 +109,22 @@ def valid_build_api?(request_body)
 end
 
 def valid_android_mapping_api?(request_body)
-  assert_equal($api_key, request_body['apiKey'])
+  valid_mapping_api?(request_body)
   assert_not_nil(request_body['proguard'])
-  assert_not_nil(request_body['appId'])
-  assert_not_nil(request_body['versionCode'])
-  assert_not_nil(request_body['buildUUID'])
-  assert_not_nil(request_body['versionName'])
 end
 
-# TODO can avoid duplication
 def valid_android_ndk_mapping_api?(request_body)
-  assert_equal($api_key, request_body['apiKey'])
+  valid_mapping_api?(request_body)
   assert_not_nil(request_body['soSymbolFile'])
+end
+
+def valid_android_ndk_mapping_api?(request_body)
```

Comment:
Seems to be exactly the same as above - copy and pasted with an edit to follow?

Reply:
Updated the method name.